app/components/friendship-page/component.js

```diff
@@ -21,6 +23,20 @@ export default Ember.Component.extend({
     return this.get('userTransactions').sortBy('createdAt').reverse()
   }.property('userTransactions.[]'),
 
+  userSettledTransactions: function() {
+    let relatedObjectId = this.get('friendship.friendshipDataId')
+
+    let settledTransactions = this.get('store').filter('transaction', t => {
+      return t.get('relatedObjectId') == relatedObjectId &&
+      t.get('relatedObjectType') == 'Friendship' && t.get('is_settled') == true
+    })
+    return settledTransactions
+  }.property('friendship.friendshipDataId'),
+
+  sortedUserSettledTransactions: function() {
+    return this.get('userSettledTransactions').sortBy('createdAt').reverse()
+  }.property('userTransactions.[]'),
```

Comment:
`sortedUserSettledTransactions` should be computed off of `userSettledTransactions` and not `userTransactions`.